flams/databases/setup.py

```diff
@@ -32,7 +32,7 @@ class ModificationType:
         "lactylation", 1.0, [ModificationDatabase(cplmv4, "Lactylation")]
     ),
     "formylation": ModificationType(
-        "formulation", 1.0, [ModificationDatabase(cplmv4, "Formylation")]
+        "formylation", 1.0, [ModificationDatabase(cplmv4, "Formylation")]
```

Comment:
there's also a typo in HMGylation. please make an Enum that will store all those names. it's very error-prone to pass them around as strings